src/sonic-config-engine/portconfig.py

```diff
@@ -246,53 +246,13 @@ def parse_platform_json_file(hwsku_json_file, port_config_file, interface_name=N
     for intf in port_dict[INTF_KEY]:
         if intf not in hwsku_dict[INTF_KEY]:
```

Comment:
Do you want to use default breakout mode here?

Reply:
Yes.